src/operations/ActionParameters.ts

```diff
@@ -34,7 +34,7 @@ export class ActionParametersUtility {
             appName: core.getInput(Inputs.APP_NAME, {"required": true}),
             useStagingDeployment: core.getInput(Inputs.USE_STAGING_DEPLOYMENT, {"required": true}).toLowerCase() == "true",
```

Comment:
required: false

Should we add default value: true?

Reply:
Follow Azure Pipeline Task implementation. It has default true defined in action.yml right now so I think required can be both true or false.